dashboard/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -20,11 +22,14 @@ class ErrorBoundary extends React.Component<IErrorBoundaryProps, IErrorBoundaryS
   public render() {
     const { error: stateError } = this.state;
     const { error: propsError } = this.props;
-    if (propsError) {
-      return <Alert theme="danger">An error occurred: {propsError.message}</Alert>;
-    }
-    if (stateError) {
-      return <Alert theme="danger">An error occurred: {stateError.message}</Alert>;
+    const err = propsError || stateError;
+    if (err) {
+      return (
+        <Alert theme="danger">
+          An error occurred: {err.message}.{" "}
+          <CdsInlineButton onClick={this.props.logout}>Logout</CdsInlineButton>
```

Comment:
It looks like this change would mean that all errors caught by the boundary will now have a logout button? Why would that be wanted?